app/assets/javascripts/views/story_view.js

```diff
@@ -499,6 +512,20 @@ module.exports = FormView.extend({
       );
     }
 
+    const attachments = this.$('.story-attachments')[0];
+    if(attachments) {
+      ReactDOM.render(
+        <StoryAttachment
+          name={'attachments'}
+          isReadonly={this.isReadonly()}
+          files={this.model.get('documents')}
+          attachinaryProps={() => this.makeAttachinaryProps()}
```

Comment:
I need to bind the function to the story view class because the function makeAttachinaryProps access the model like this `this.model.get('documents')`